packages/stripe_platform_interface/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: stripe_platform_interface
 description: Platform interface for stripe sdk
-version: 2.0.2
+version: 2.0.3
```

Comment:
Can you revert this? Version bump is not needed we use Melos to bump and publish the versions